.github/BlockedBillboard.java

```diff
@@ -0,0 +1,60 @@
+

```

Comment:
Add comments on:
* How many minutes did you spend
* Test results
* difficult? easy? which part?